src/lib/uTop_main.cppo.ml

```diff
@@ -283,6 +283,9 @@ let rec map_items unwrap wrap items =
       | Outcometree.Osig_modtype (name, _)
       | Outcometree.Osig_value (name, _, _) ->
         (name, Outcometree.Orec_not)
+#if OCAML_VERSION >= (4, 03, 0)
+      | Outcometree.Osig_ellipsis -> assert false
+#endif
```

Comment:
Beware here. I don't know how to fix since I mostly don't know utop's internals.